news/_posts/2022-10-30-playtest-20221030.md

```diff
@@ -0,0 +1,87 @@
+    <source src="{{ '/images/news/20221031-transient.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-transient.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>Nuke incoming!</figcaption>
+</figure>
+
+The fairly recent Dune film has brought many players to Dune 2000 and a decently sized multiplayer community was born. With the feedback from the community Dune 2000 has received a number of balance and polish changes.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-dune.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-dune.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-dune.png' | relative_url }}" width="700" />
+  </video>
+</figure>
+
+Tiberian Sun has also received its fair share of features and bug fixes. This time around the highlight is tilting on slopes.
+
+<figure>
+  <video width="700" autoplay loop muted>
+    <source src="{{ '/images/news/20221031-slope.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20221031-slope.ogv' | relative_url }}" type="video/ogg">
+    <img src="{{ '/images/news/20221031-slope.png' | relative_url }}" width="700" />
+  </video>
+  <figcaption>OpenRA now supports tilting on slopes!</figcaption>
+</figure>
+
+<br>
+
+Some of the more visible new features and improvements include:
+ * Measures against spamming the lobby and in-game chat
+ * The ability to mute players
+ * Support for disabling the chat in replays
+ * New Red Alert missions
+ * An ingame encyclopedia in Dune 2000
+
+Notable changes for mappers and modders include:
+ * Automatic map reloading in the background
+ * A change to the way speed is calculated for ground units
+ * A new map format which includes the map preview (minimap) in the map versioning (map hash)
+ * Loaders for more audio formats (e.g., mp3 and ogg), and formats used by the remasters
+ * Many linting and documentation updates
+
+As well as some other changes worth mentioning:
+ * Toolchain and changes to support the .NET 6 runtime
+ * A massive amount of upgrades to both ingame and client UI
+ * Many more hotkeys became rebindable
+ * Substantial progress towards localisation
+
+All in all, 615 pull requests were merged since last release and 552 issues were closed bringing the total number of open tickets down by 141.
+
+This post covered just a small subset of all the changes and improvements added since the last release. We encourage anybody interested in finding out more to read the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog). Head on over to the [download page]({{ '/download' | relative_url }}) to install the new playtest and let us know what you think in the comments below, on [our forum](https://forum.openra.net/), [community Discord server](https://discord.openra.net) or on [GitHub](https://github.com/OpenRA/OpenRA/issues). Happy playtesting!
```

Comment:
https://github.com/OpenRA/OpenRA/wiki/Changelog/1e2499c01b75fc76018dfe96ce915379557b3716 is the full changelog link.